Extension/src/LanguageServer/client.ts

```diff
@@ -1772,6 +1778,9 @@ export class DefaultClient implements Client {
             cppSettingsScoped["default"] = { systemIncludePath: cppSettingsResourceScoped.get("default.systemIncludePath") };
         }
 
+        const otherSettingsWorkspace: OtherSettings = new OtherSettings(this.RootUri);
```

Comment:
This shouldn't be named "workspace", because existing code like workspaceOtherSettings is being used for that. It could be WorkspaceFolder and the Global one could be Workspace.